### PR TITLE
explain error message on scaffolding

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

File: aspnetcore/includes/RP/model4Win.md [hunk 1]
```diff
@@ -7,6 +7,13 @@
   ```console
   dotnet aspnet-codegenerator razorpage -m Movie -dc MovieContext -udl -outDir Pages\Movies --referenceScriptLibraries
   ```
+  
+If you get the error:
+  ```
+No executable found matching command "dotnet-aspnet-codegenerator"
+  ```
+
+Open a command window in the project directory (The directory that contains the *Program.cs*, *Startup.cs*, and *.csproj* files).
```
Comment (rachelappel) [inline aspnetcore/includes/RP/model4Win.md]: "Open a command window in the project directory" and do what? Just open the command window and no further action?

Comment (Rick-Anderson) [reply aspnetcore/includes/RP/model4Win.md]: You have to read it context.  see https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/model?branch=pr-en-us-4806 and search for Open a command window

Comment (Rick-Anderson) [reply aspnetcore/includes/RP/model4Win.md]: Maybe I should make it 
Open **the** command window in the proj dir

**Comment by Rick-Anderson** [comment]
@rachelappel  thanks for the review. I could have been more clear on that but I don't think too many folks hit it. I've been getting one a week tho. Hopefully that's enough to get them past the problem.

**Pull request merged**